litmus-portal/frontend/src/components/Tabs/index.tsx

```diff
@@ -24,7 +24,7 @@ const StyledTab = withStyles((theme) =>
   createStyles({
     root: {
       textTransform: 'none',
-      color: 'rgba(0,0,0,0.5)',
+      color: theme.palette.tabsComponentColor,
```

Comment:
https://github.com/litmuschaos/litmus/pull/2120/files#r496476443

Reply:
similar usage as mention in the above comment.
https://github.com/litmuschaos/litmus/pull/2120#discussion_r496723353